recipes/cm-unicode/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cm-unicode" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://kent.dl.sourceforge.net/project/{{ name }}/{{ name }}/{{ version }}/{{ name }}-{{ version }}-ttf.tar.xz
+  sha256: 2609c14450f42d0bcd40203900afcb1d693521a9b24a18c65e14b6b0585ff150
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  host:
+    - matplotlib
```

Comment:
Why does this require matplotlibe to install?

Reply:
Because I copy the fonts in the matplotlib fonts folder, so it makes no sense to install this package without matplotlib

Reply:
Well, you don't  depend on matplotlib at runtime.... you are just putting them in a matplotlib directory.  You don't need matplotlib for that.  In fact, I would recommend against it since it will increase CI time and bandwidth to build the package.  You might need to depend on python to get the right site packages?

Is there a  more global place that fonts typically are installed to that can then you can add soft-links in the matplot lib directory?

Reply:
`${Prefix}/fonts` ?

Reply:
Lots of things I am seeing (eg. https://askubuntu.com/a/3706/416923) indicate it would be `${PREFIX}/share/fonts/`

Reply:
I was looking at https://anaconda.org/anaconda/font-ttf-dejavu-sans-mono/

Reply:
Ahh yeah, if that is what is used, let's keep with that.  I would be great to have that documented too

Reply:
We actually have to decide if we are packaging the fonts for generic use (which may need a patch on matplotlib to find them) or if we are packing them only for mpl. While the former is more elegant I have to confess that I am OK with the practicality of the latter. We all already have fonts in our systems and this package would be only to satisfy mpl.

Reply:
I tired: 
```
import matplotlib.font_manager
matplotlib.font_manager.findSystemFonts(fontpaths=None, fontext='ttf')
```
And at least on my system `$PREFIX/fonts` is already included.